firebase-auth/package.json

```diff
@@ -8,20 +8,20 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/TeamHubApp/firebase-kotlin-multiplatform-sdk.git"
+    "url": "git+https://github.com/GitLiveApp/firebase-kotlin-multiplatform-sdk.git"
```

Comment:
wrong url

Reply:
This isn't there anymore it in the root gradle->subprojects

Reply:
Done in all package.json's